### DISCUSSION TITLE
能否考虑下通过Github Release分发安装包呢？

### DISCUSSION THREAD

**Discussion by Seele0oO** [Meloong-Git/PCL#4273] [讨论]
目前通过直接在存储库里放文件，这对于git维护很不友好吧。

**Comment by allMagicNB** [comment]
应该是为了方便更新……放 Code 里可以直链下载，Release 下载更复杂。

**Reply by FloatSheep** [reply]
我猜不是这个原因，GitHub 提供了不需要 Token 的方式获取最新版本

`https://github.com/user/repo/releases/latest/download/filename`

更换一下，即可以直接访问 `https://github.com/FloatSheep/surl/releases/latest/download/surl.exe` 进行下载

**Reply by Deep-Dark-Forest** [reply]
不是的，FCL 启动器用的就是从 Release 下载

**Comment by hejiehao** [comment]
https://github.com/Hex-Dragon/PCL2/issues/4043#issuecomment-2169731286
#3249
